tools/cli_install.py

```diff
@@ -85,6 +85,10 @@ def _get_tempfilename(a_dir):
 def _mark_executable(path):
     os.chmod(path, 0o755)
 
+def install_cli_from_dir(src_dir, write_dir):
+    src_file = os.path.join(src_dir, get_cli_filename())
+    install_cli(src_file, write_dir)
+
```

Comment:
The weird thing about python is that the development community has confused PEP8, the style guide for python core, with good python style.  So there's a bunch of stuff in there that's super sensible for all projects, and then a bunch of other stuff that's completely arbitrary. 

It's fine that a particular project has arbitrary style enforcement, but this is an example of one of the style choices that doesn't make universal sense.  For some code that's primarily well-factored libraries with clearly defined responsibility, the vertical density may not matter as much, because you consider things in an isolated way typically.  In a script it just seems pretty counterproductive to me.